### ISSUE TITLE
Silent NPE when receiving test push notification

### ISSUE THREAD

**Issue by calvarez-ov** [CleverTap/clevertap-android-sdk#66]
I've set up a campaign to show an alert (popup) when the app sends a certain event. I'm testing out this campaign using the "send test notification" feature.

My device receives the notification, and it does show the alert. But, at some point, I was troubleshooting an issue with the debugger, and found that an exception is caught and silently ignored.

The exception is:
```
java.lang.NullPointerException: Attempt to invoke virtual method 'void android.app.Fragment.setArguments(android.os.Bundle)' on a null object reference
    at com.clevertap.android.sdk.CleverTapAPI.checkExistingInAppNotifications(CleverTapAPI.java:1901)
    at com.clevertap.android.sdk.CleverTapAPI.activityResumed(CleverTapAPI.java:1871)
    at com.clevertap.android.sdk.CleverTapAPI.onActivityResumed(CleverTapAPI.java:415)
    at com.clevertap.android.sdk.CleverTapAPI.onActivityResumed(CleverTapAPI.java:385)
    at com.clevertap.android.sdk.ActivityLifecycleCallback$1.onActivityResumed(ActivityLifecycleCallback.java:53)
    at android.app.Application.dispatchActivityResumed(Application.java:239)
    at android.app.Activity.onResume(Activity.java:1343)
    at androidx.fragment.app.FragmentActivity.onResume(FragmentActivity.java:455)
    at com.myapp.MyActivity.onResume(MyActivity.kt:62)
```

The code which causes the exception:
```
    private void checkExistingInAppNotifications(Activity activity) {
        final boolean canShow = canShowInAppOnActivity();
        if (canShow) {
            if (currentlyDisplayingInApp != null) {
                Fragment inAppFragment = activity.getFragmentManager().getFragment(new Bundle(), currentlyDisplayingInApp.getType());
                if (getCurrentActivity() != null) {
                    FragmentTransaction fragmentTransaction = getCurrentActivity().getFragmentManager().beginTransaction();
                    Bundle bundle = new Bundle();
                    bundle.putParcelable("inApp", currentlyDisplayingInApp);
                    bundle.putParcelable("config", config);
                    inAppFragment.setArguments(bundle);  <------ HERE
```
`inAppFragment` is `null` in this case.

This exception is caught in `CleverTapAPI.onActivityResumed()`:
```
        for (String accountId : CleverTapAPI.instances.keySet()) {
            CleverTapAPI instance = CleverTapAPI.instances.get(accountId);
            try {
                if (instance != null) {
                    instance.activityResumed(activity);
                }
            } catch (Throwable t) {
                Logger.v("Throwable - "+t.getLocalizedMessage()); <--- HERE
            }
        }
```

I'm not sure what the impact of this is. The app doesn't crash, but it looks like an unexpected code path that might be fixable.

**Comment by darshanclevertap** [comment]
@calvarez-ov Thank you for pointing this out, my team is looking into this. I'll keep you updated

**Comment by piyush-kukadiya** [comment]
@calvarez-ov This issue has been fixed in v3.8.0
